README.md

```diff
@@ -11,7 +11,7 @@ To play, follow one of the installation guides below. After that, you can just n
 You might also want to `git pull` to ensure that your git history is up-to-date. Otherwise, you may end up playing the wrong game for the day. The program doesn't do this automatically because I didn't want it to make any changes to the file system.
 
 ### Installation from source (recommended)
-With any v1.21 or greater of [Golang](https://go.dev/) you can easily install from source:
+With any v1.22 or greater of [Golang](https://go.dev/) you can easily install from source:
```

Comment:
I _think_ this requires 1.22 because the `go.mod` specifies `go 1.22.3`

Reply:
Go 1.21 onwards will automatically install the toolchain specified in the `go.mod`, which is why that's the minimum.

I'm planning on rewording this anyway, so this doesn't need to block.